### DISCUSSION TITLE
Where can I put worker on serwist?

### DISCUSSION THREAD

**Discussion by tranvanhieu01012002** [serwist/serwist#165] [Q&A]
I am using this package for nextjs PWA, it is good

But I'm having a requirement with worker,

I found a document
[https://ducanh-next-pwa.vercel.app/docs/next-pwa/custom-worker](custom-worker)

I try it, but it doesn't work
so, I put it on ```sw.js```
but I want to a specific file for worker code
[https://github.com/shadowwalker/next-pwa/blob/master/examples/web-push/worker/index.js](worker/index.js)

Please support me

**Comment by DuCanhGH** [comment]
@tranvanhieu01012002 

`next-pwa`'s custom worker concept does not exist in Serwist. Just write the code into your SW source. You can use the static `import` declaration to import bindings from other modules into your service worker.
